<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>339</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>NanumSquare</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Ani24 Downloader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="dn_btn">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>80</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="dn_progress2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>761</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="dn_link">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>761</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumSquare</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>https://ani24do.com/ani_list/2858.html</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumSquare</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ani24 Video Link</string>
    </property>
   </widget>
   <widget class="QLabel" name="dn_fname">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>721</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumSquare</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dn_name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>721</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumSquare</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="dn_progress1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>761</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="Copy">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Made by Morgan_KR, Ani24_Downloader</string>
    </property>
   </widget>
   <widget class="QLabel" name="dn_status">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>270</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Downloader Started~!</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSet_Directory"/>
    <addaction name="actionInfo"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSet_Directory">
   <property name="text">
    <string>Set Directory</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
